tests/conftest.py

```diff
@@ -75,6 +75,86 @@ def mock_getBuild_two():
     }
 
 
+@pytest.fixture
+def mock_getBuild_three():
```

Comment:
Calling this `mock_getBuild_building` would be easier to read.

Reply:
Alternatively, you could just reuse one of the existing fixtures and then change the state manually in your test to avoid creating these fixtures. We do this in other places already I believe.